app/src/main/java/io/branch/branchster/SplashActivity.java

```diff
@@ -58,10 +59,16 @@ public void onInitFinished(BranchUniversalObject branchUniversalObject, LinkProp
                 if (branchUniversalObject == null) {
                     proceedToAppTransparent();
                 }
+                /* In case the clicked link has $android_deeplink_path the Branch will launch the MonsterViewer automatically since AutoDeeplinking feature is enabled.
+                 * Launch Monster viewer activity if a link clicked without $android_deeplink_path
+                 */
                 else if (!branchUniversalObject.getMetadata().containsKey("$android_deeplink_path")) {
```

Comment:
The live version of iOS is not putting deep link path keys in the links. Here's another one that doesn't have `+clicked_branch_link` erroneously put in the link https://bnc.lt/ALMc/iLhneh1Qav. Again, I'd recommend that we change the deep link key to `monster`.